htdocs/accountancy/class/accountancyexport.class.php

```diff
@@ -163,13 +163,13 @@ public function exportNormal($objectLines) {
 
 		foreach ( $objectLines as $line ) {
 			// Std export
-			$date = dol_print_date($line->doc_date, $conf->global->ACCOUNTING_EXPORT_DATE);
+			$date = dol_print_date($line->doc_date, '%d/%m/%Y');
```

Comment:
hard coded format is forbidden. Must use "day" or "dayhour"
also why loosing option ACCOUNTING_EXPORT_DATE ?